interview-questions.md

```diff
@@ -8,44 +8,45 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: Super would be used in a ruby class when you want to access the instance variables from the parent class.
 
-Researched answer:
+Researched answer:  The super method calls the initialize method in the parent class. After creating a subclass, the subclass can use the super method to borrow the instance variables from the parent class to access them.
 
 2. What is a gem?
 
-Your answer:
+Your answer: Gem is a library or snippet of codes.
 
-Researched answer:
+Researched answer: Gem is a library that is managed by a package manager called RubyGems. Gems provide specific functionality or tools that are implemented to ruby applications. 
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: Relational database is like a document that contains information displayed in rows and columns. There are other kinds of databases such as object oriented databases or non-relational databases. 
 
-Researched answer:
+Researched answer: Relational database is a type of database that stores data and organizes them as a table with rows and columns. The
+columns have names and data type which is called schema. The rows are identified with primary keys or unique identifiers. There are other kinds of databases such as object oriented, non-relational, hierarchical, network, NoSQL and many others.
 
 4. What are primary keys? Why purpose does a primary key serve?
 
-Your answer:
+Your answer: Primary key is a way of uniquely identifying rows in a relational database. A primary key is used to make sure that there are no duplicate values.
 
-Researched answer:
+Researched answer: Primary key is a column in a relational database that is used as an unique identifier for the rows in the table. It is used to prevent duplicate values and makes sure each row in the table is unique. 
 
 5. What are the five HTTP verbs? What is each verb's corresponding CRUD action?
 
-Your answer:
+Your answer: The five HTTP verbs are GET, DELETE, PATCH, POST and PUT. GET's corresponding CRUD action is read. DELETE's corresponding CRUD action is delete. PATCH's corresponding CRUD action is update. POST's corresponding CRUD action is create. PUT's corresponding CRUD action is update.
 
-Researched answer:
+Researched answer: The five HTTP verbs include GET, DELETE, PATCH, POST and PUT. The corresponding CRUD action for the verbs respectively are read, delete, update, create and update.
```

Comment:
These are great answers!  I really like that you expanded on your definitions in the researched answers!  😎